Branch-SDK/Branch-SDK/BNCStrongMatchHelper.m

```diff
@@ -97,27 +95,28 @@ - (void)presentSafariVCWithBranchKey:(NSString *)branchKey {
 
     [urlString appendFormat:@"&sdk=ios%@", SDK_VERSION];
     
-    SFSafariViewController *safController = [[SFSafariViewController alloc] initWithURL:[NSURL URLWithString:urlString]];
-    safController.delegate = self;
-    
-    UIViewController *windowRootController = [[UIViewController alloc] init];
-    
-    self.secondWindow = [[UIWindow alloc] initWithFrame:CGRectZero];
-    self.secondWindow.rootViewController = windowRootController;
-    [self.secondWindow makeKeyAndVisible];
-    [self.secondWindow setAlpha:0];
-
-    [windowRootController presentViewController:safController animated:NO completion:NULL];
-}
-
-- (void)safariViewController:(SFSafariViewController *)controller didCompleteInitialLoad:(BOOL)didLoadSuccessfully {
-    [self.secondWindow.rootViewController dismissViewControllerAnimated:NO completion:NULL];
-
-    if (didLoadSuccessfully) {
-        [BNCPreferenceHelper preferenceHelper].lastStrongMatchDate = [NSDate date];
+    Class SFSafariViewControllerClass = NSClassFromString(@"SFSafariViewController");
+    if (SFSafariViewControllerClass) {
+        id safController = [[SFSafariViewControllerClass alloc] initWithURL:[NSURL URLWithString:urlString]];
+        
```

Comment:
actually, since we don't conform to the protocol, probably don't need it. looks like we don't rely on any of their callbacks either.